README.md

```diff
@@ -43,6 +43,12 @@ cargo xtask build --cpioz=$CPIOZ
 This generates a "Debug" binary in the file
 `target/x86_64-oxide-none-elf/debug/phbl`.
 
+**Note**: The precoded `gld` linker in `x86_64-oxide-none-elf.json` is short
+for GNU ld, to work around clashes on BSD-alikes, where ld would be the BSD
```

Comment:
This actually isn't quite true; we run this mostly on illumos, which is a System V derivative and uses the Solaris linker by default. I'd just remove the bit about BSD and say something like,

"Linking `phbl` requires using the GNU linker.

By default Oxide's build systems install GNU ld as `gld`.  If your
installation calls it something else, set the
[environment variable](https://doc.rust-lang.org/cargo/reference/environment-variables.html)
`CARGO_TARGET_X86_64_OXIDE_NONE_ELF_LINKER=ld` to use whatever
name is appropriate on your system."

Reply:
Rewritten - thank you!